monai/transforms/lazy/functional.py

```diff
@@ -24,19 +26,196 @@
     kwargs_from_pending,
     resample,
 )
+from monai.transforms.traits import LazyTrait
+from monai.transforms.transform import MapTransform
 from monai.utils import LazyAttr, look_up_option
 
-__all__ = ["apply_transforms"]
+__all__ = ["apply_pending_transforms", "apply_pending_transforms_in_order", "apply_pending"]
 
 __override_keywords = {"mode", "padding_mode", "dtype", "align_corners", "resample_mode", "device"}
 
 
-def apply_transforms(
-    data: torch.Tensor | MetaTensor, pending: list | None = None, overrides: dict | None = None, **kwargs: Any
+def _log_pending_info(
```

Comment:
This doesn't appear to be covered by tests, is this a concern at the moment or something we'll come back to?

Reply:
That is surprising. I'll have a quick look. It should be covered by some tests already though.

Reply:
@ericspod I think this PR is ready to merge, do you have any other comments?

Thanks.

Reply:
`_log_pending_info` is definitely hit by tests in `test_compose.py`